doc/content/sensor-edition/beta-welcome/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "WELCOME, WE ARE EXCITED!"
```

Comment:
I suggest `Welcome, we are excited to have you onboard!`

Reply:
Done